README.md

```diff
@@ -1,6 +1,79 @@
 # terraform-aws-vpc
 
-Terraform module that defines a VPC with public/private subnets across multiple AZs with Internet Gateways
+Terraform module that defines a VPC with Internet Gateway.
+
+
+## Usage
+
+* Quick start example:
+
+```terraform
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  name      = "${var.name}"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+}
+```
+
+* Full example with [`terraform-aws-dynamic-subnets`](https://github.com/cloudposse/terraform-aws-dynamic-subnets.git):
+
+```terraform
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  name      = "${var.name}"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+}
+
+module "dynamic_subnets" {
+  source = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=master"
+
+  availability_zones = "${var.availability_zones}"
+  namespace          = "${var.namespace}"
+  name               = "${var.name}"
+  stage              = "${var.stage}"
+  region             = "${var.region}"
+  vpc_id             = "${module.vpc_id}"
+  igw_id             = "${module.igw_id}"
+  cidr_block         = "10.0.0.0/16"
```

Comment:
Let's use `${module.vpc.igw_id}`.
If the `cidr_block` is out of VPC CIDR block range, an error is raised.